src/execution_plan/ops/op_value_hash_join.c

```diff
@@ -32,33 +32,54 @@ static bool _record_islt(Record l, Record r, uint idx) {
 #define RECORD_SORT_ON_ENTRY(a, b) (_record_islt((*a), (*b), idx))
 
 // Performs binary search, returns the leftmost index of a match.
-static int64_t _binarySearchLeftmost(Record *array, int join_key_idx, SIValue v) {
-	int64_t recordCount = array_len(array);
-	int64_t pos;
-	int64_t left = 0;
-	int64_t right = recordCount;
+static bool _binarySearchLeftmost(int64_t *idx, Record *array, int join_key_idx, SIValue v) {
```

Comment:
I think this signature should accept `array_len` just for consistency with `Rightmost`.